apps/web/src/components/Shared/Login/WalletSelector.tsx

```diff
@@ -57,6 +57,7 @@ const WalletSelector: FC<Props> = ({ setHasConnected, setHasProfile }) => {
   };
 
   const handleSign = async () => {
+    let keepModal = false;
```

Comment:
An alternative to adding a bool flag would be to add `setShowAuthModal(false);` in three places. I tried it but I thought it looked less clean and less future proof.